src/ui/story.js

```diff
@@ -7,7 +7,7 @@ You may obtain a copy of the BSD License at
 https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENSE.txt
 */
 
-/* global fluid */
+/* global fluid, sjrk */
```

Comment:
You can add "sjrk" to the eslint config so that you don't need to specify it as a global in all the files.

Reply:
I began to work on this, but then realized it affects nearly every file (as `fluid` is [also specified](https://github.com/fluid-project/eslint-config-fluid/blob/041cd0f476fc02d0293914962c14912750e753f5/.eslintrc.json#L7) in the `.eslintrc.json` for `eslint-config-fluid` itself. Due to the already cluttered nature of this PR, I have decided this is out of scope and described the work under the freshly-filed [SJRK-417](https://issues.fluidproject.org/browse/SJRK-417)